pinot-common/src/main/java/org/apache/pinot/common/exception/QueryException.java

```diff
@@ -152,7 +156,7 @@ public static void setMaxLinesOfStackTrace(int maxLinesOfStackTracePerFrame) {
     QUERY_SCHEDULING_TIMEOUT_ERROR.setMessage("QuerySchedulingTimeoutError");
     SERVER_RESOURCE_LIMIT_EXCEEDED_ERROR.setMessage("ServerResourceLimitExceededError");
     EXECUTION_TIMEOUT_ERROR.setMessage("ExecutionTimeoutError");
-    DATA_TABLE_DESERIALIZATION_ERROR.setMessage("DataTableSerializationError");
+    DATA_TABLE_SERIALIZATION_ERROR.setMessage("DataTableSerializationError");
```

Comment:
I've also changed this because it seems like an error. `DATA_TABLE_DESERIALIZATION_ERROR` was in fact set 2 lines below

Reply:
Already fixed by @Jackie-Jiang in d8d1e92cb9fd252b00b44f303c77e22592c47ecd